### ISSUE TITLE
Snapshot of page with WebGL causes persistent discrete GPU switch

### ISSUE THREAD

**Issue by dstillman** [zotero/zotero#1308]
https://forums.zotero.org/discussion/67470/zotero-causing-battery-drain-on-macbook-pro-w-discrete-graphics

> From testing on our end, we've been able to reproduce this when saving a snapshot of a page that uses WebGL, such as https://get.webgl.org/. Flash, HTML5 video, or framed PDFs don't seem to trigger it. Unfortunately, none of the Mozilla framework options (in Advanced prefs -> Config Editor) to disable WebGL, such as webgl.disabled, seem to help here, and the GPU stays on until close even though the hidden browser used for saving the snapshot is cleaned up properly.

I have a local branch with some changes to how we handle hidden browsers, so we can see if that makes a difference. Not sure what else to try here other than a new Firefox version.

**Comment by dstillman** [comment]
@tnajdek, are you seeing this with a PDF link?

**Comment by tnajdek** [comment]
No, saving this pdf does not trigger switch to high performance gpu.
